client/webserver/site/src/css/market.scss

```diff
@@ -63,24 +63,33 @@ table.ordertable {
   margin-bottom: 10px;
 
   th {
-    font-size: 14px;
     font-family: 'demi-sans', sans-serif;
   }
 
-  td {
-    font-size: 14px;
-  }
-
   td,
   th {
-    font-size: 14px;
+    font-size: 13px;
     padding: 3px 0;
   }
 
   tr {
     border-bottom: 1px solid #b3b3b3;
   }
 
+  thead,
+  tr {
+    display: table;
+    width: 100%;
+    table-layout: fixed;
+  }
+
+  tbody {
+    display: block;
+    width: 100%;
+    overflow: scroll;
```

Comment:
Although I had suggested `overflow: auto`, I think we should just not set overflow here at all since the parent elements already are `stylish-overflow` class, which makes a scroll bar with consistent appearance across browsers.